recipes-core/iwinfo/iwinfo_git.bbappend

```diff
@@ -0,0 +1,71 @@
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# Subject to the terms and conditions of this license, each copyright holder
+# and contributor hereby grants to those receiving rights under this license
+# a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable
+# (except for failure to satisfy the conditions of this license) patent license
+# to make, have made, use, offer to sell, sell, import, and otherwise transfer
+# this software, where such license applies only to those patent claims, already
+# acquired or hereafter acquired, licensable by such copyright holder or contributor
+# that are necessarily infringed by:
+#
+# (a) their Contribution(s) (the licensed copyrights of copyright holders and
+# non-copyrightable additions of contributors, in source or binary form) alone;
+# or
+#
+# (b) combination of their Contribution(s) with the work of authorship to which
+# such Contribution(s) was added by such copyright holder or contributor, if,
+# at the time the Contribution is added, such addition causes such combination
+# to be necessarily infringed. The patent license shall not apply to any other
+# combinations which include the Contribution.
+#
+# Except as expressly stated above, no rights or licenses from any copyright
+# holder or contributor is granted under this license, whether expressly, by
+# implication, estoppel or otherwise.
+#
+# DISCLAIMER
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# NOTE
+#
+# This is part of a management middleware software package called MMX that was developed by Inango Systems Ltd.
+#
+# This version of MMX provides web and command-line management interfaces.
+#
+# Please contact us at Inango at support@inango-systems.com if you would like to hear more about
+# - other management packages, such as SNMP, TR-069 or Netconf
+# - how we can extend the data model to support all parts of your system
+# - professional sub-contract and customization services
+#
+################################################################################
+DEPENDS += "lua5.1"
```

Comment:
To build iwinfo I had to remove 0001-Makefile-LDFLAGS-set-liblua5.1-for-lua-lib.patch
from SRC_URI:
`SRC_URI_remove += "file://0001-Makefile-LDFLAGS-set-liblua5.1-for-lua-lib.patch"`
Am I doing something wrong during my build?

Reply:
I had to do the same
I see such changes in branch `feature/PPM-1154-mmx-on-rdkb` of `meta-prplmesh` layer
https://gitlab.com/prpl-foundation/prplmesh/meta-prplmesh/-/blob/feature/PPM-1154-mmx-on-rdkb/recipes-core/iwinfo/iwinfo_git.bbappend

And i agree that removing patch must be done in `meta-mmx` instead